src/Settings/settings.json

```diff
@@ -1208,5 +1208,22 @@
         "name": "WakaTime/Enable"
       }
     ]
+  },
+  {
+    "name": "Display Stopwatch",
+    "type": "bool",
+    "default": false
+  },
+  {
+    "name": "Toggle Stopwatch On Tab Switch",
+    "desc": "Start/Stop stopwatch on tab switch",
+    "type": "bool",
+    "default": false,
+    "tip": "When switching to a different tab, automatically start the stopwatch\non the current tab and pause the stopwatch on the previous tab.",
```

Comment:
```suggestion
    "tip": "When switching to a different tab, automatically start the stopwatch on the current tab and pause the stopwatch on the previous tab.",
```

Reply:
The tooltip doesn't auto-wrap. Break the line to prevent it from being too long.

Reply:
Sorry my bad 🤦🏻‍♂️